index.html

```diff
@@ -1,25 +1,53 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Code Journal</title>
   <link rel="icon" href="images/favicon.ico">
-  <link
-    rel="stylesheet"
-    crossorigin="anonymous"
+  <link rel="stylesheet" crossorigin="anonymous"
     href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
     integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==">
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Heebo:wght@500&display=swap" rel="stylesheet">
 </head>
+
 <body>
   <header>
+    <h1>Code Journal</h1>
   </header>
   <main>
+    <div class="container" data-view="entry-form">
+      <form action="">
+      <h1 class="form-header">New Entry</h1>
+      <div class="row">
+        <div class="column-half ">
+          <img class= "image-to-change" src="images\placeholder-image-square.jpg" alt="Default Blank Image">
+        </div>
+          <div class="column half">
+            <div class="image-box-submit column-half"><label for="image-URL">Image URL</label>
+              <input type="url" name="url" id="image-URL" required="required">
+            </div>
+            <div class="title-field column-half"><label for="Title">Title</label>
+              <input type="text" name="text" id="Title" required="required">
+            </div>
+          </div>
+      </div>
+      <div class="column-full"><label for="Notes">Notes</label>
+        <textarea name="Notes" id="Notes" cols="40" rows="15" required></textarea>
```

Comment:
You should make the `id` value here lowercase. Make sure to update the label `for` attribute as well!